### PR TITLE
Conditions 605 Add additional unit tests to Add Disabilities page

### PR DESCRIPTION
**Pull request by williamphelps13**
## Summary
- Summarize the changes that have been made to the platform: added, organized, and refined Add Disabilities unit testing
- Which team do you work for, does your team own the maintenance of this component? Conditions Team and Yes

## Related issue(s)
[Add additional unit tests to Add Disabilities page #605](https://app.zenhub.com/workspaces/contention-classification-640a24db5dcd08457cebbc79/issues/gh/department-of-veterans-affairs/vagov-claim-classification/605)

### CODE REVIEW

**Comment by bacitracin** [comment]
Another test you could add is that on page load focus moves to the "2 of 5" heading

File: src/applications/disability-benefits/all-claims/tests/components/ComboBox.unit.spec.jsx [hunk 1]
```diff
@@ -293,25 +293,24 @@ describe('ComboBox Component', () => {
       expect(listbox).to.have.length(0);
     });
 
-    // TODO: Throwing error "An error was thrown inside one of your components, but React doesn't know what it was"
-    // it.only('should select item using Enter and make listbox empty', () => {
-    //   const searchTerm = 'k';
-    //   const searchResults = fullStringSimilaritySearch(searchTerm, items);
-    //   const { getByRole, getByTestId } = render(<ComboBox {...props} />);
+    it('should select item using Enter and make listbox empty', () => {
```
Comment (bacitracin) [inline src/applications/disability-benefits/all-claims/tests/components/ComboBox.unit.spec.jsx:296]: Was the problem the `.only`?

**Review by bacitracin — APPROVED** [review]
Had a comment about another test you can add but not blocking

**Review by pjhill — APPROVED** [review]
LGTM -- thanks for reinstating that unit test!

**Pull request merged**